Makefile

```diff
@@ -41,7 +41,8 @@ tests: init $(TEST_OUT_FILE)
 
 init:
 	@mkdir -p $(BUILD_DIR) $(BUILD_DIR)/dfas build/tests build/lib
-	@$(foreach TEST_CASE, $(TEST_CASES), @python Extras/Scripts/listTestFiles.py tests/$(TEST_CASE)/ tests/include/$(TEST_CASE)TestFiles.h  tests/src/$(TEST_CASE)TestFiles.cpp $(TEST_CASE);)
+	@$(foreach TEST_CASE, $(TEST_CASES), @python Extras/Scripts/listTestFiles.py tests/$(TEST_CASE)/ tests/include/$(TEST_CASE)TestFiles.h  tests/src/$(TEST_CASE)TestFiles.cpp $(TEST_CASE));
```

Comment:
The semi colon was in the right place - make file statements don't have semi colons